container/Makefile

```diff
@@ -52,10 +52,11 @@ URL_ncbi-amrfinderplus := https://depot.galaxyproject.org/singularity/ncbi-amrfi
 URL_blast := https://depot.galaxyproject.org/singularity/blast:2.14.0--hf3cf87c_0
 URL_bracken := https://depot.galaxyproject.org/singularity/bracken:2.8--py39hc16433a_0
 URL_bwakit := https://depot.galaxyproject.org/singularity/bwakit:0.7.17.dev1--hdfd78af_1
-URL_chewbbaca := https://depot.galaxyproject.org/singularity/chewbbaca:3.3.2--pyhdfd78af_0
+URL_chewbbaca := https://depot.galaxyproject.org/singularity/chewbbaca:3.3.10--pyhdfd78af_0
```

Comment:
Just make sure to test this with ecoli and saureus to make sure the output hasn't changed too much.

Reply:
Backrolling it since it since I have no resources to validate it. 
It should be looked into updating chewbacca when the opportunity arises.

Reply:
Yeah there are a few things that need to be updated!